docs/package_integration/guide.md

```diff
@@ -5,7 +5,7 @@
 This guide is intended for developers integrating applications with UDS (Unicorn Delivery Service). It provides an overview of the integration process, key considerations, and resources to ensure a smooth integration.
 
 Integrating a Package fundamentally means:
-1. Creating a repository `uds-package-<name>`
+1. Creating a repository `uds-package-<name>` from [uds-package-template](https://github.com/defenseunicorns/uds-package-template)
```

Comment:
For legacy projects like LeapfrogAI, I assume this step doesn't apply as long as we integrate UDS common and package template capabilities from upstream sources? (E.g., UDS tasks)

Reply:
Correct.

1/ that document is a guide to help people navigate vs rules that must be followed
2/ if legacy projects want to obtain bronze/silver/gold badging they'd have to meet those rules. (many of which come for "free" by using the template) However, even the badging specifies **should** regarding template vs **must**

Reply:
That would be my assumption as well - SWF actually still hasn't used the template repo even for brand new packages like `uds-package-renovate` :eyes: